app/services/request_password_reset.rb

```diff
@@ -16,14 +17,22 @@ def perform
   private
 
   def send_reset_password_instructions
+    irs_params = {
+      email: email,
+    }
+
     if Throttle.new(user: user, throttle_type: :reset_password_email).throttled_else_increment?
       analytics.throttler_rate_limit_triggered(throttle_type: :reset_password_email)
+      irs_attempts_api_tracker.forgot_password_email_rate_limited(**irs_params)
     else
       token = user.set_reset_password_token
       UserMailer.reset_password_instructions(user, email, token: token).deliver_now_or_later
 
       event = PushNotification::RecoveryActivatedEvent.new(user: user)
       PushNotification::HttpPush.deliver(event)
+
+      irs_params[:success] = true
+      irs_attempts_api_tracker.forgot_password_email_sent(**irs_params)
```

Comment:
same comment about inlining, I think it would be clearer
```suggestion
      irs_attempts_api_tracker.forgot_password_email_sent(email: email, success: true)
```